web/source/kmwcallback.ts

```diff
@@ -1125,14 +1125,14 @@ namespace com.keyman {
             case 'tablet':
             case 'phone':
             case 'desktop':
-              if(this.keymanweb.util.activeDevice.formFactor != constraint) {
+              if(this.keymanweb.util.device.formFactor != constraint) {
```

Comment:
We need to match off the actual device, not the simulated hardware device which says 'desktop'. We are still a mobile device in that situation.